pyproject.toml

```diff
@@ -5,7 +5,7 @@ requires = [
     "cmake",
     "numpy<1.24",
 ]
-build-backend = "setuptools.build_meta"
+#build-backend = "setuptools.build_meta"
```

Comment:
According to these doc, removing the explicit line does not do anything. Can we please revert this? Explicit is better than implicit.

https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/

Reply:
Nevermind, it took a bit to understand that `--no-use-pep517` requires that this line is commented out. Which is very strange.